packages/terra-application-links/src/tabs/ApplicationTabs.scss

```diff
@@ -62,12 +77,12 @@
   .tab-menu-display {
     align-items: center;
     background-color: transparent;
-    background-image: var(--terra-application-tabs-background-image, linear-gradient(to bottom, #007ee0, #007ee0));
     background-position: var(--terra-application-tabs-background-position, top-left);
     background-repeat: no-repeat;
-    background-size: var(--terra-application-tabs-background-size, 0 4px);
+    background-size: var(--terra-application-tabs-background-size, 0 5px);
     border: 0;
-    border-right: var(--terra-application-tabs-border-right, 1px solid #dedfe0);
+    border-right: var(--terra-application-tabs-border-right, 1px solid #3a67a4);
+    box-shadow: var(--terra-application-tabs-box-shadow, 1px 0 1px #1d98da);
```

Comment:
Looks like the check themeable variables webdriver tests need to be updated with the changes made here. Some need removed and quite a few need added